hack/istio/multicluster/kiali-prepare-remote-cluster.sh

```diff
@@ -35,6 +35,7 @@ DEFAULT_PROCESS_REMOTE_RESOURCES="true"
 DEFAULT_REMOTE_CLUSTER_CONTEXT="west"
 DEFAULT_REMOTE_CLUSTER_NAME=""
 DEFAULT_REMOTE_CLUSTER_NAMESPACE="kiali-access-ns"
+DEFAULT_SERVER=""
```

Comment:
This script has some specifically named variables here. "Home Cluster" is where Kiali is, "Remote Cluster" is the other external cluster. Using "Server" here is ambiguous (home cluster server? remote cluster server? name? url?)

Since this is customer-facing script (users will use this to create kiali remote cluster secrets), let's make this more specific and consistent with the rest of the names. Also, based on reading what this variable is, this is a URL not a server name ("Server" could imply just a name, but this is really a URL).

```suggestion
DEFAULT_REMOTE_CLUSTER_URL=""
```